appmonitoring/ts/src/CertificateGenerator.ts

```diff
@@ -232,6 +233,53 @@ export class CertificateManager {
         return caCertificate.verify(certificate);
     }
 
+    private static IsValidCertificate(operationId: string, mwhcCaBundle: string, webhookCertData: WebhookCertData, clusterArmId: string, clusterArmRegion: string): boolean {
+        try {
+            forge.pki.certificateFromPem(mwhcCaBundle);
+            forge.pki.certificateFromPem(webhookCertData.caCert);
+            forge.pki.certificateFromPem(webhookCertData.tlsCert);
+            forge.pki.privateKeyFromPem(webhookCertData.tlsKey);
+            return true;
+        } catch (error) {
+            logger.error('Error occured while trying to validate certificates!', operationId, this.requestMetadata);
+            logger.error(JSON.stringify(error), operationId, this.requestMetadata);
+            logger.SendEvent("CertificateValidationFailed", operationId, null, clusterArmId, clusterArmRegion, true, error);
+            return false;
+        }
+    }
+
+    private static async CheckCertificateJobStatus(kubeConfig: k8s.KubeConfig, operationId: string, clusterArmId: string, clusterArmRegion: string): Promise<boolean> {
+        kubeConfig.loadFromDefault();
```

Comment:
I thought we decided this wasn't needed?

Reply:
This is definitely needed :)

Reply:
I thought we decided we would only call `loadFromDefault` once immediately after initialization? This appears to be one of those extra calls that we don't need, correct?

Reply:
ah you mean only this line, somehow I thought you were asking if this entire method was needed :D